modules/olm-refresh-subs.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="olm-refresh-subs_{context}"]
+= Refreshing failing subscriptions
+
+In Operator Lifecycle Manager (OLM), if you subscribe to an Operator that references images that are not accessible on your network, you can find jobs in the `openshift-marketplace` namespace that are failing with the following errors:
+
+.Example output
+[source,terminal]
+----
+ImagePullBackOff for
+Back-off pulling image "example.com/openshift4/ose-elasticsearch-operator-bundle@sha256:6d2587129c846ec28d384540322b40b05833e7e00b25cca584e004af9a1d292e"
+----
+
+.Example output
+[source,terminal]
+----
+rpc error: code = Unknown desc = error pinging docker registry example.com: Get "https://example.com/v2/": dial tcp: lookup example.com on 10.0.0.1:53: no such host
+----
+
+As a result, the subscription is stuck in this failing state and the Operator is unable to install or upgrade.
+
+You can refresh a failing subscription by deleting the existing subscription, deleting the new cluster service version (CSV) and install plan, then recreating the subscription. OLM then forcefully reinstalls the correct version of the Operator.
+
+.Procedure
+
+. Get the names of the `Subscription`, `ClusterServiceVersion`, and `InstallPlan` objects from the namespace where the Operator is installed. For example:
```

Comment:
No need to get the `InstallPlan`, it will be deleted automatically after the Subscription removed.